homeassistant/components/isy994.py

```diff
@@ -132,6 +134,20 @@ def _categorize_programs() -> None:
                         PROGRAMS[component].append(program)
 
 
+def _categorize_weather() -> None:
+    """Categorize the ISY994 weather data."""
+    global WEATHER_NODES
```

Comment:
By the by, after pushing this PR to dev, I was hoping to open a PR against the master. I have a couple of hotfixes. This one and the event decorators are broken.